src/EPR.Calculator.API/Builder/Summary/TwoCCommsCost/TwoCCommsCostColumnIndex.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace EPR.Calculator.API.Builder.Summary.TwoCCommsCost;
+
+public class TwoCCommsCostColumnIndex
+{
+    public const int Value = 203;
```

Comment:
For one property do we need separate class? Can we add this in the same class or all sections indexes into one class which looks good. If other devs also follow this single class pattern then we end up with multiple classes. 
What do you say?